content/_policy/rules-of-use._fr.md

```diff
@@ -85,13 +85,13 @@ Si nous soupçonnons une fraude sur votre compte, nous pouvons verrouiller votre
-Outre la validation des preuves d'identité que vous nous fournissez, nous vérifions que vous êtes bien cette personne. Nous évaluons l'appareil utilisé pour accéder à Login.gov, l'identité vérifiée et votre comportement pendant la vérification de l'identité. Nous collectons des informations à partir de l'appareil pour effectuer cette évaluation. Vos informations personnelles sont stockées d'une manière qui empêche Login.gov de les lire. Cependant, Login.gov peut toujours évaluer les informations concernant votre appareil. Nous expliquons plus en détail comment nous utilisons vos informations dans notre [politique de confidentialité]({% link _policy/index._fr.md %}).
+Outre la validation des preuves d'identité que vous nous fournissez, nous vérifions que vous êtes bien cette personne. Nous évaluons l'appareil utilisé pour accéder à Login.gov, l'identité vérifiée et votre comportement pendant la vérification de l'identité. Nous collectons des informations à partir de l'appareil pour effectuer cette évaluation. Vos informations personnelles sont stockées d'une manière qui empêche Login.gov de les lire. Cependant, Login.gov peut toujours évaluer les informations concernant votre appareil. Nous expliquons plus en détail comment nous utilisons vos informations dans notre [Politique de Confidentialité](/fr/policy/).
 
 ## 5. Résiliation
 
-Vous pouvez résilier votre compte à tout moment par l'intermédiaire de votre profil de compte. En outre, en cas de fraude ou d'autres violations des présentes règles d'utilisation, nous pouvons révoquer l'accès à votre compte. Dans ce cas, nous continuerons à protéger les informations de votre compte conformément à notre [politique de confidentialité]({% link _policy/index._fr.md %}) et à notre avis sur le système d'archivage [ici](https://www.federalregister.gov/documents/2022/11/21/2022-25420/privacy-act-of-1974-notice-of-a-modified-system-of-records).
+Vous pouvez résilier votre compte à tout moment par l'intermédiaire de votre profil de compte. En outre, en cas de fraude ou d'autres violations des présentes règles d'utilisation, nous pouvons révoquer l'accès à votre compte. Dans ce cas, nous continuerons à protéger les informations de votre compte conformément à notre [Politique de Confidentialité](/fr/policy/) et à notre [](https://www.federalregister.gov/documents/2022/11/21/2022-25420/privacy-act-of-1974-notice-of-a-modified-system-of-records)[avis sur le système d’enregistrements](https://www.federalregister.gov/documents/2022/11/21/2022-25420/privacy-act-of-1974-notice-of-a-modified-system-of-records).
```

Comment:
* Same `{% link %}` issue
* There's a second, empty link tag here for System of Records that should be removed

```suggestion
Vous pouvez résilier votre compte à tout moment par l'intermédiaire de votre profil de compte. En outre, en cas de fraude ou d'autres violations des présentes règles d'utilisation, nous pouvons révoquer l'accès à votre compte. Dans ce cas, nous continuerons à protéger les informations de votre compte conformément à notre [Politique de Confidentialité]({% link _policy/index._fr.md %}) et à notre [avis sur le système d’enregistrements](https://www.federalregister.gov/documents/2022/11/21/2022-25420/privacy-act-of-1974-notice-of-a-modified-system-of-records).
```